### ISSUE TITLE
Investigate & fix - ERR_HTTP_HEADERS_SENT

### ISSUE THREAD

**Issue by Nicanor008** [Nicanor008/shoman-api#19]
On the console, at some point, we get ` Cannot set headers after they are sent to the client`. 
<img width="842" alt="Screenshot 2021-02-03 at 05 55 05" src="https://user-images.githubusercontent.com/25298492/106691483-a2033480-65e4-11eb-9f7a-145f5f446f5a.png">

**Comment by neymarjimoh** [comment]
This has something to do with the middlewares. I had these errors when I used the middlewares for testing, after removing them it disappeared. I mentioned it earlier that I couldn't use the middlewares. I thought I was the only one having this issue

**Comment by neymarjimoh** [comment]
I'd check more on this

**Comment by Nicanor008** [comment]
> This has something to do with the middlewares. I had these errors when I used the middlewares for testing, after removing them it disappeared. I mentioned it earlier that I couldn't use the middlewares. I thought I was the only one having this issue

Yeah, it has something to do with the middlewares so of, see if you can enhance them to be more performant and not displaying any errors

**Comment by neymarjimoh** [comment]
No problem I will @Nicanor008
